### PR TITLE
Language updates

### LINKED ISSUE #467: Add separate question about landlord being a person or business before asking for the landlord's name
**The usability audit pointed out that this question was challenging for multiple participants because the question does not match what the subquestion asks first:**

Screenshot 2024-08-06 at 2 37 03 PM
Screenshot 2024-08-06 at 2 41 20 PM

**Usability feedback:**
Screenshot 2024-08-06 at 2 39 16 PM

### LINKED ISSUE #480: Update phrasing for question about landlord being the city or state?
Screenshot 2024-08-07 at 12 39 43 PM

### LINKED ISSUE #487: Increase retaliation information box to avoid scroll bar
Screenshot 2024-08-07 at 2 43 40 PM

### LINKED ISSUE #489: Add subquestion explanation of what needs to be entered when asked "How much is your rent?"
Screenshot 2024-08-08 at 10 32 09 AM

### LINKED ISSUE #495: Update language on "Review your documents before you sign" question to ensure it is clear to the user what the next steps are
Screenshot 2024-08-08 at 12 30 08 PM

### LINKED ISSUE #512: Remove links to specific FAQs on Mass Legal Help site that don't actually lead to these questions, just the main info page
These links are not useful anymore because they can't link to the specific related FAQ accordions on Mass Legal Help's [Affidavit of Indigency if you can't afford Massachusetts court costs](https://www.masslegalhelp.org/courts-hearings-language-rights/courts-massachusetts/affidavit-indigency-if-you-cant-afford) page.

Screenshot 2024-08-21 at 12 29 56 PM

### LINKED ISSUE #513: Update language for landlord name to reflect user answering
Certain fields for landlord being a business or individual and landlord name are not responsive to the user filling out the form and instead default to the user being the tenant.

### LINKED ISSUE #515: Make it clear that UpToCode is specific to MA

### PR DESCRIPTION
**Pull request by KelizaMcGuire**
Fixes language issues found during the usability audit and user testing and updates urls throughout the interview.

Fixed #513, Fixed #489, Fixed #480, Fixed #467, Fixed #512, Fixed #487, Fixed #515, Fixed #495

- [X] I have manually tested the interview with these changes
- [X] I have requested a review from Quinten

### CODE REVIEW

File: docassemble/HousingCodeChecklist/data/questions/customized_screens.yml [hunk 1]
```diff
@@ -121,21 +127,42 @@ if: |
...
       Deeds for your city or town.
+      % else:
+      **Okay**. You can still fill in this form, but the tenant will need
+      to write in the name of their landlord on every form before they deliver
+      it. You may be able to find the name of the tenant's landlord by visiting the Registry
+      of Deeds for the tenant's city or town.
+      % endif
     show if: ll_name_unknown
-  - Is your landlord an individual or a business?: other_parties[0].person_type
+  - ${"Is your landlord an individual or a business?" if person_answering == "tenant" else "Is the tenant's landlord an individual or a business?"}: other_parties[0].person_type
```
Comment (nonprofittechy) [inline docassemble/HousingCodeChecklist/data/questions/customized_screens.yml:165]: Slight preference to write this out longhand vs using a 1-line if statement/ternary (it's a bit easier to get the translator to handle correctly):
```suggestion
  - label: |
      % if person_answering =="tenant":
      Is your landlord an individual or a business?
      % else:
      Is the tenant's landlord an individual or a business?
      %endif
    field: other_parties[0].person_type
```

File: docassemble/HousingCodeChecklist/data/questions/housing_code_interview.yml [hunk 2]
```diff
@@ -236,6 +236,10 @@ question: |
 subquestion: |
   Need help to get repairs or stop your landlord from harassing you? Pick a task.
   
+  <div class="alert alert-danger h-20 mh-20" role="alert">
+    <p class="h6 alert-danger"><i class="fa-solid fa-triangle-exclamation fa-xl"></i> In order to use UpToCode your rental unit must be located in Massachusetts.</p>
+  </div>
```
Comment (nonprofittechy) [inline docassemble/HousingCodeChecklist/data/questions/housing_code_interview.yml:241]:
```suggestion
  Need help to get repairs or stop your **Massachusetts** landlord from harassing you? Pick a task.
  
```

Just thinking to make this warning clear but not as aggressive.

**Review by nonprofittechy — APPROVED** [review]
Some minor suggestions--slight improvement to translatability. I think we shouldn't have a danger triangle on the front page though. Review, add the commits, and merge if those changes look good.

**Comment by KelizaMcGuire** [comment]
@nonprofittechy All look good to me! That makes sense, I wasn't sure how intense we wanted to be in our references to MA only language. I definitely prefer your suggestion.

**Pull request merged**